### ISSUE TITLE
[BUG] Both light and dark themes fail basic web standards

### ISSUE THREAD

**Issue by weedySeaDragon** [eslint/config-inspector#137]
This tool is helpful -- thanks!

But the theme is _terrible_.  It doesn't even meet the _minimum_ web accessibility standards -- the contrast is **WAY TOO LOW** both the light and dark versions. C'mon.  It's 2025.   These standards have been around for a loooong time. There's a ton of information about it and some great free tools out there. (Obviously it's totally fine if someone personally likes a theme that fails the standards -- we're all different. But when you publish something, you need to follow guidelines so your project works for the largest number of people. )

This was noted in issue #64 but nothing has been done about it.  So I'm raising it again and providing some info and links to free tools (which, TBH, you should already know about and be using):

1. [Web Accessibility In Mind (WebAIM)](https://webaim.org/): The site for this group is _great for getting a good, fast, and practical intro to implementing web accessibility.  They have practical, easy to use checklists and free tools to check any website. (Including the Config Inspector of course!)
    - [Standards Checklist for Developers](https://webaim.org/standards/wcag/checklist):  This page has practical lists for WCAG standards. (You can pick which version of the standard and what level.)
    - [1.4.3 Minimum Contrast](https://webaim.org/standards/wcag/checklist#sc1.4.3) This is the entry for WCAG 2, level AA.  (Clicking on the heading will take you to the official standard.)
    - ### [Free tool for evaluating a web page: WAVE](https://wave.webaim.org/) 
        Under ["Resources" on their site](https://webaim.org/resources/), they have some free tools, including **_free browser extensions._**  This is what  I've used for years. It's straightforward and you can get a quick summary or as many details as you can handle. (Yes, I made that large on purpose.) 
        It has an easy-to-use tool where you can click on the color and it will show you the contrast level with the background/foreground. It has a standard color-picker type UI so you can adjust it and see the results and ratio immediately.

There are many other free browser tools and sites that will evaluate web pages. I wanted to share this one because it's easy and reliable. NAYY

I'm willing to help fix the contrast issues.  Not sure if just another theme should be used or if you want to tweak the colors in the existing theme.
